src/test/regress/spec/isolation_dml_vs_repair.spec

```diff
@@ -31,7 +31,7 @@ session "s1"
 setup
 {
     DEALLOCATE all;
-    TRUNCATE test_dml_vs_repair;
+    DELETE FROM test_dml_vs_repair;
```

Comment:
why we need this change? Probably using `master_create_distributed_table ` causes this, maybe convert it to `create_distributed_table` ?

Reply:
I tried doing that, but some of the testing commands fails. Since it is not TRUNCATE that is being tested here, but the repair functionality, and TRUNCATE is merely used to reset the table, changing TRUNCATE > DELETE FROM was the minimal change I could introduce to fix the test.

Reply:
The diifference is likely thee replication factor. `create_distributed_table ` can use replication_factor 
```sQL
    SELECT master_create_distributed_table('test_dml_vs_repair', 'test_id', 'hash');
    SELECT master_create_worker_shards('test_dml_vs_repair', 1, 2);
```

To 
```SQL
   SET citus.shard_replication_factor TO 2;
    SELECT create_distributed_table('test_dml_vs_repair', 'test_id', 'hash', shard_count:=1);
```

Reply:
It was exactly that. It works now 👍